src/main/java/Main.java

```diff
@@ -137,7 +139,18 @@ private static void rename(Client client) throws SftpException {
           break;
         case 6:
           System.out.println("Rename remote directory/file...");
+          try {
+            client.displayRemoteFiles();
+            out.println("Enter the original file name: ");
```

Comment:
To keep the switch statement clean and for the sake of consistency (change dir and create dir methods prompt from within), I'd move the filename prompts to the method implementation in the client class and just call the method here without args. Other than that, it works perfectly!

Reply:
Perfect, thanks! 👌